tools/lsp/preview.rs

```diff
@@ -106,6 +108,7 @@ pub fn finish_parsing(ok: bool) {
 }
 
 pub fn config_changed(
+    show_preview_ui: bool,
     style: &str,
     include_paths: &[PathBuf],
     library_paths: &HashMap<String, PathBuf>,
```

Comment:
maybe put all of this in a struct so you don't have to pass them around all the time.